src/proto/robot_config.proto

```diff
@@ -18,7 +18,110 @@ package bowler.robot_config;
 
 option java_multiple_files = true;
 option java_package = "com.commonwealthrobotics.proto.robot_config";
+//next 2
+// a center of mass object
+message CenterOfMass{
+  Pose centroid = 1;
+  double mass_kg = 2;
+}
 
-// Next: 1
+// next 3
+// a file location of a provider of some depency to be injected
+message GitFile {
+  string url = 1;
+  string branch = 2;
+  string file = 3;
+}
+// next 10
+message LimbType{
```

Comment:
Why are we classifying the limbs at all? What does this help us do?

Reply:
yeah, the BodyCOntroller will heed to know what limbs are to be used for what. the only difference between a leg and an arm is this tag saying so. Its not an enum because swapping out controllers will want to use a limb differently, like legs with wheels on the end have a driving controller, and a walking controller as options, which will use the limb differently